etc/env/example.env

```diff
@@ -197,3 +201,81 @@ DATASETS_SERVICE_PORT=8880
 AUTHORIZATION_SERVICE_VERSION=v0.0.1-alpha
 AUTHORIZATION_SERVICE_HOST=0.0.0.0
 AUTHORIZATION_SERVICE_PORT=7000
+
+
+# keycloak service
+CANDIG_AUTH_CONTAINER_NAME=candigauth.local
+KEYCLOAK_VERSION=9.0.2
+KC_ADMIN_USER=
+KC_ADMIN_PW=
+KC_TEST_USER=
+KC_TEST_PW=
+KC_TEST_USER_TWO=
+KC_TEST_PW_TWO=
+KC_REALM=candig
+KC_CLIENT_ID=cq_candig
+KC_LOGIN_REDIRECT_PATH=/auth/login
+## KC_SECRET= ###(generated in setup.sh)
+## KC_CLIENT_ID_64= ###(generated in setup.sh)
+## KC_PUBLIC_KEY= ###(generated in setup.sh)
+KEYCLOAK_SERVICE_PUBLIC_PORT=8081
+KEYCLOAK_SERVICE_CONTAINER_PORT=8081
+KEYCLOAK_SERVICE_HOST=0.0.0.0
+
+KEYCLAOK_SERVICE_PUBLIC_PROTO=http
```

Comment:
does there need to be different variable for public protocol vs internal protocol

Reply:
yes,
within the cluster, at least in a dev environment, the communication between containers is over http, but https outside the cluster